1.9/administration/logging/aggregating/logging-api.md

```diff
@@ -0,0 +1,132 @@
+---
+post_title: Logging API 
+menu_order: 1
+---
+
+# dcos-go/dcos-log - HTTP server for journal events
+
+# REST API
+#### Endpoints:
+- `/range/` get a range of logs matching the request query.
```

Comment:
this is a standalone service endpoints for the cluster it should have a prefix:
every node exposes these endpoints:
`/system/v1/logs/v1/range/`
`/system/v1/logs/v1/stream/`
`/system/v1/logs/v1/fields/<field>`

only agent nodes expose the task log endpoints:
`/system/v1/logs/v1/{range|stream}/framework/{framework_id}/executor/{executor_id}/container/{container_id}`

only masters expose a special proxy endpoints:
`/system/v1/agent/<agent_id>/logs/v1/{range|stream}` that will be reverse proxied to an appropriate agent based on `<agent_id>` parameter